aw-sync/src/main.rs

```diff
@@ -60,7 +60,29 @@ struct Opts {
 enum Commands {
     /// Daemon subcommand
     /// Starts aw-sync as a daemon, which will sync every 5 minutes.
-    Daemon {},
+    Daemon {
+        /// Use advanced sync mode
+        /// (automatically enabled when any advanced options are provided)
+        #[clap(long)]
+        advanced: bool,
```

Comment:
Can be removed now

Reply:
Necessary for syncing all buckets by default, though I can remove it and use something like `--buckets *` as an alternative or just sync all buckets by default, resorting to syncing specific buckets when it is specified.